### PR TITLE
Unit test speed up

### LINKED ISSUE #526: Shorten unit test duration
Currently, we have one unit test which takes significantly more time to run than all others:

```
118.13s call     pyteal/ast/router_test.py::test_method_config
```

@ahangsu believes it can be refactored to take less time.

### PR DESCRIPTION
**Pull request by ahangsu**
Relying on randomness, randomly take some testcases from full permutation to test method config.

Test duration from 100+s to about 3s.

Closes #526 

No CHANGELOG for changes are not reflected to lib usage.

### CODE REVIEW

**Comment by barnjamin** [comment]
@ahangsu is this good to go? I'd love to get it merged in if so :patpray:

**Review by barnjamin — APPROVED** [review]

File: pyteal/ast/router_test.py [hunk 1]
```diff
@@ -320,12 +355,10 @@ def test_method_config():
         for oc in ON_COMPLETE_CASES
         if str(oc) != str(pt.OnComplete.ClearState)
     ]
+    focg = FullOrderCombinationGen(list(pt.CallConfig), len(ON_COMPLETE_CASES))
```
Comment (michaeldiamant) [inline pyteal/ast/router_test.py:358]: @ahangsu I opened an _optional_ child PR (https://github.com/algorand/pyteal/pull/569) intending to make debugging simpler.  Can you have a look?

Feel welcomed to keep as is.

**Review by michaeldiamant — APPROVED** [review]
@ahangsu Thanks for taking the time to vet the issue.  Looks reasonable to me - moving one step closer to property-based testing!

File: pyteal/ast/router_test.py (hunk 1 above)
Comment (ahangsu) [reply pyteal/ast/router_test.py]: merged

**Pull request merged**